packages/modular-scripts/src/__tests__/start-app.ts

```diff
@@ -113,6 +113,7 @@ export async function startApp(
   // starts above, but don't want to wait until the
   // process itself finishes.
   return {
+    // eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
Why is this promise "misused" and not "floating"? Asking because I think we can silence the "floating" error without eslint comments, by just prepending `void`.